dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-details.test.ts

```diff
@@ -44,7 +44,6 @@ describe('get-utilisation-details', () => {
         secondTfmFacility.facilitySnapshot.ukefFacilityId = secondFacilityId;
 
         jest.spyOn(TfmFacilitiesRepo, 'findByUkefFacilityIds').mockResolvedValue([firstTfmFacility, secondTfmFacility]);
-
```

Comment:
can the line be added again

Reply:
not a blocker

Reply:
Good spot - maybe worth doing this in the next PR, in the interests of the deadline / clearing the PR backlog

Reply:
Yeah just saw this also, just put it back!

Reply:
Excellent reply timings here 😂